tasks/main.yml

```diff
@@ -1,235 +1,58 @@
 ---
-
 - name: Set version specific variables
   include_vars: "{{ item }}"
   with_first_found:
-    - "{{ ansible_distribution }}-{{ ansible_distribution_version }}.yml"
-    - "{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
+    - "{{ ansible_distribution }}_{{ ansible_distribution_version }}.yml"
+    - "{{ ansible_distribution }}_{{ ansible_distribution_major_version }}.yml"
```

Comment:
why this change?

Reply:
I prefer underscores over dashes, but it appears that dashes are more desirable in filenames. Changing this.

Reply:
Oh. Looks like I have underscores supported per here: https://github.com/oasis-roles/meta_standards#naming-things

Reply:
Good point - I did not realize it applied to files as well. I will discuss this with OASIS colleagues, because I always had the impression that dashes are more idiomatic in file names, but maybe we will change this for consistency.
Anyway, I think that such stylistic changes do not belong to this PR. Please keep it focused to the topic of SLES and Ubuntu support.